src/main/resources/sam.conf

```diff
@@ -211,6 +211,11 @@ azureServices {
         publisher = "thebroadinstituteinc1615909626976"
         authorizedUserKey = authorizedTerraUser
       }
+      {
```

Comment:
Only change relevant to ticket. Other changes are needed to fix broken Sam swat tests